runtime/restart/monitor/monitor.go

```diff
@@ -220,11 +220,13 @@ func (m *monitor) monitor(ctx context.Context) ([]change, error) {
 		}
 		desiredStatus := containerd.ProcessStatus(labels[restart.StatusLabel])
 		task, err := c.Task(ctx, nil)
-		if err != nil && desiredStatus == containerd.Stopped {
+		if err != nil {
```

Comment:
This changes the logic some, is there a case this was trying to account for when `Task` may return not found and the switch statement will handle it when desired state is running?

Reply:
I modified the logic to be the same as [before](https://github.com/containerd/containerd/commit/9503d7219e2adb5906f4c396f3f47e5eb45272c5#diff-a9d90e82d0fee932f0437ab217a8ef14e4a56b4be02321de5193de0ad98e20f9L239-L249)